oauth2client/service_account.py

```diff
@@ -108,6 +117,25 @@ def serialization_data(self):
             'private_key': self._private_key_pkcs8_text
         }
 
+    @classmethod
+    def from_json(cls, s):
+        s = _from_bytes(s)
+        data = json.loads(s)
```

Comment:
Same comment here about just doing this in one line instead of reassigning your parameter.